autogen.sh

```diff
@@ -5,7 +5,31 @@ set -xe
 type autoreconf || exit 1
 type pkg-config || exit 1
 
-ctags_files=`make -f makefiles/list-translator-input.mak --no-print-directory`
+gen_bsdmakefile() {
+	set +xe
+	echo -e '.DONE:\n\t@echo "Please use GNU Make (gmake) to build this project"\n.DEFAULT:\n\t@echo "Please use GNU Make (gmake) to build this project"' > BSDmakefile
```

Comment:
Can't you generate the 2 rules at once? like
```Makefile
.DONE .DEFAULT:
	@echo "Please use GNU Make (gmake) to build this project"
```

Reply:
Also, would it be handy to forward the call to GNU Make automatically?  If
```makefile
.DONE .DEFAULT:
	@echo "Please use GNU Make (gmake) to build this project"
	make $@
```
is enough it seems handy.  OK, maybe it's trickier because we'd like to forward some of the `MAKEFLAGS` but they are not all compatible between GNU and BSD make (e.g. I know NetBSD make adds a `-J` option GNU make doesn't understand to `MAKEFLAGS` when the user uses `-j`).  Just a suggestion in case it wasn't already considered.